core/src/main/resources/openapi-specs/eth2/signing/paths/high_watermark.yaml

```diff
@@ -0,0 +1,24 @@
+get:
+  tags:
+    - 'High Watermark'
+  summary: 'The High Watermark epoch and slot applicable to all validators'
+  description: 'Returns the uint64 epoch and slot of the high watermark. Signing of attestations or blocks are prevented at or beyond this high watermark'
```

Comment:
Think we should make it clear that if there is no high watermark then an empty result is returned

Reply:
done